src/Shaders/default.vertex.fx

```diff
@@ -171,8 +173,9 @@ void main(void) {
 #include<shadowsVertex>[0..maxSimultaneousLights]
 
 #ifdef VERTEXCOLOR
-	// Vertex color
 	vColor = color;
+#elif INSTANCESCOLOR

```

Comment:
ok stopping here but make sure to review all the code duplication

Reply:
I'll keep this one too!

Reply:
There were traps to see if you were following :)

_(or me being a dumb)_